helm-frontend/src/routes/Run.tsx

```diff
@@ -45,10 +45,10 @@ export default function Run() {
   const [instances, setInstances] = useState<Instance[]>([]);
   const [stats, setStats] = useState<Stat[]>([]);
   const [displayPredictionsMap, setDisplayPredictionsMap] = useState<
-    DisplayPredictionsMap | undefined
+    undefined | { [key: string]: { [key: string]: DisplayPrediction[] } }
```

Comment:
nit: `undefined | Record<string, Record<string, DisplayPrediction[]>>` is a little cleaner

Reply:
Fixed